start-from-zero/plus.js

```diff
@@ -0,0 +1,11 @@
+
+var plus = document.querySelector('.plus-sign');
```

Comment:
I just wanted to store the .plus-sign class into a variable and later on I wanted to attach a click event listener so I could toggle between '+' and '-' on click. I will fix this today.